install.lsp

```diff
@@ -155,7 +165,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE."
 (LOAD "overwrite.fas")
 (DELETE-FILE "overwrite.fas")
 (DELETE-FILE "overwrite.lib")
-;(load "platform.shen")
+;(load "platform.shen") ; TODO - should this remain commented-out?
```

Comment:
This can be uncommented, so that it works the same as the SBCL port.

I'm not sure, but I think in the later versions released by Mark SBCL became the official CL port, so the CLisp one may have lagged a bit behind.